docs/guide/n3iwue-installation.md

```diff
@@ -0,0 +1,147 @@
+<!-- Google tag (gtag.js) --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-JETJ7TJ805"></script> <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-JETJ7TJ805'); </script>
+
+# Installing N3IWUE 
+
+In this demo we will practice:
+
+- Installing N3IWUE
+- Configuring free5GC and N3IWUE
+- Running N3IWUE to connect free5GC
+
+## 1. Install N3IWUE VM
+
+Repeat the steps of cloning `free5gc` VM from the base VM, create a new VM for the N3IWUE:
+
+- You can refer to [Installing free5gc](https://free5gc.org/guide/3-install-free5gc/) to install free5gc VM.
+- Name the VM `n3iwue`, and create new MAC addresses for all network cards.
+- Make sure the VM has internet access and can log in using SSH.
+- Change the hostname to `n3iwue`.
+- Make the Host-only network interface have static IP address `192.168.56.103`.
+- Reboot the n3iwue VM, as well as the free5gc VM.
+- You can ping `192.168.56.101` from the n3iwue VM, and also `ping 192.168.56.103` from the free5gc VM.
+
+## 2. Install N3IWUE
+
+Search “free5gc/n3iwue” on the web, and get the [web site](https://github.com/free5gc/n3iwue).
```

Comment:
A little weired on this sentence. 
Suggestion:
Go to [N3IWUE GitHub Repo](https://github.com/free5gc/n3iwue).